app/app.go

```diff
@@ -580,11 +606,12 @@ func NewStrideApp(
 	// - transfer
 	// - base app
 	recordsStack := recordsmodule.NewIBCModule(app.RecordsKeeper, transferIBCModule)
+	ratelimitStack := ibcratelimitmodule.NewIBCModule(recordsStack, app.IbcRateLimitKeeper)
```

Comment:
can we actually flip the two parameters here? I much prefer your version with the module as the first parameter, but we're unable to change the ordering of the ICA stacks and it makes it a little easier to read if they all follow the same format
```suggestion
	ratelimitStack := ibcratelimitmodule.NewIBCModule(app.IbcRateLimitKeeper, recordsStack)
```

Reply:
Ok. let me flip them.